wolfssl/tests/ping_pong.rs

```diff
@@ -192,18 +193,21 @@ async fn server<S: SockIO>(sock: S, protocol: Protocol) {
     println!("[Server] Finished");
 }
 
+#[test_case(Protocol::DtlsClientV1_2, Protocol::DtlsServerV1_2)]
```

Comment:
`test_case` will derive the test name from this here you get `dtls::protocol_dtlsclientv1_3_protocol_dtlsserverv1_3_expects`.

You can specificy a particular name with

```
[test_case(Protocol::DtlsClientV1_2, Protocol::DtlsServerV1_2; "client_1.2_server_1.3")]
```
if we want to make it a little more readable.

Reply:
Oh nice! will update it